README.md

```diff
@@ -20,7 +20,6 @@ This framework was born out of necessity, hardships and trouble in the developme
 ```
 npm install etherlime
```

Comment:
Do we need -g here?

Reply:
No not really, it is developers choice how to use it. Both will work

Reply:
If installed without -g the run command etherlime deploy returns following - bash: etherlime: command not found

Reply:
much like truffle. you'd need to run it from `node_modules/.bin/etherlime`

Reply:
I'll change it to `npm install etherlime [-g]`